SPCTR-3497: Fix - Content Timeline static styles getting applied to other nested blocks (#3616)
brainstormforce/ultimate-addons-for-gutenberg: README.md

* Fix: Icon List - Resolved an issue wherein the vertical layout alignment would not appear as intended in the editor's tablet view.
* Fix: FAQ - Resolved an issue wherein the block content padding would appear differently in the editor and the front-end.
* Fix: Global - Resolved an issue wherein the WooCommerce and custom template-styles would not load as intended in FSE theme.

> * Fix: Content Timeline - Resolved an issue wherein the static styles of Content Timeline getting applied to other nested blocks.

Reviewer: ```suggestion
* Fix: Content Timeline - Resolved an issue wherein the block's styling would be applied to other nested blocks in the editor.
```

Author: Done.
